corehq/apps/settings/forms.py

```diff
@@ -55,3 +59,167 @@ def save(self, commit=True):
         if commit:
             couch_user.save()
         return user
+
+
+class HQPhoneNumberMethodForm(PhoneNumberMethodForm):
+
+    def __init__(self, **kwargs):
+        super(HQPhoneNumberMethodForm, self).__init__(**kwargs)
+        self.helper = FormHelper()
+        self.helper.form_class = 'form form-horizontal'
+        self.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.helper.field_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.helper.layout = crispy.Layout(
+            crispy.Fieldset(
+                '',
+                'number',
+                crispy.Div(crispy.Field('method'), css_class='radio')
+            ),
+            hqcrispy.FormActions(
+                twbscrispy.StrictButton(
+                    _('Next'),
+                    css_class='btn-primary',
+                    type='submit',
+                )
+            )
+        )
+
+
+class HQDeviceValidationForm(DeviceValidationForm):
+
+    def __init__(self, **kwargs):
+        super(HQDeviceValidationForm, self).__init__(**kwargs)
+        self.helper = FormHelper()
+        self.helper.form_class = 'form form-horizontal'
+        self.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.helper.field_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.helper.layout = crispy.Layout(
+            crispy.Fieldset(
+                '',
+                'token'
+            ),
+            hqcrispy.FormActions(
+                twbscrispy.StrictButton(
+                    _("Back"),
+                    css_class='btn-default',
+                    type='submit',
+                    value='method',
+                    name="wizard_goto_step",
+                ),
+                twbscrispy.StrictButton(
+                    _('Next'),
+                    css_class='btn-primary',
+                    type='submit',
+                ),
+            )
+        )
+
+
+class HQTwoFactorMethodForm(MethodForm):
+
+    def __init__(self, **kwargs):
+        super(HQTwoFactorMethodForm, self).__init__(**kwargs)
+        self.helper = FormHelper()
+        self.helper.form_class = 'form form-horizontal'
+        self.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.helper.field_class = 'col-sm-3 col-md-4 col-lg-2'
```

Comment:
Should probably be `col-sm-9 col-md-8 col-lg-6, fields generally get more space than their labels